src/hotspot/share/opto/reg_split.cpp

```diff
@@ -569,8 +565,14 @@ uint PhaseChaitin::Split(uint maxlrg, ResourceArea* split_arena) {
 #undef NEW_SPLIT_ARRAY
 
   // Initialize to array of empty vectorsets
-  for( slidx = 0; slidx < spill_cnt; slidx++ )
-    UP_entry[slidx] = new VectorSet(split_arena);
+  // Each containing at most spill_cnt * _cfg.number_of_blocks() entries.
+  for (slidx = 0; slidx < spill_cnt; slidx++) {
+    UP_entry[slidx] = new(split_arena) VectorSet(split_arena);
+  }
+
+  // Keep track of DEFS & Phis for later passes
+  Node_List defs{split_arena, 8};
+  Node_List phis{split_arena, 16};
```

Comment:
Why do you use aggregate initialization instead of constructor invocation here?

Reply:
This will call the constructor, and has since C++11 I believe.  However, I'm clearly being inconsistent here and with the VectorSet change above. There's no reason that I picked brace initializer other than it being the 'modern' way: https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Res-list

I'll revert this change and use regular parens to be more stylistically consistent.

Reply:
Thanks for the background, I wasn't aware of that. I don't have a strong opinion but consistency in the same area would be nice.

Reply:
Please, change to normal `()`. Using '{}' is very confusing for not modern C++ experts and affects maintainability of this code.